### DISCUSSION TITLE
Use breakpoints in tw.style()?

### DISCUSSION THREAD

**Discussion by PercentageCorporation** [jaredh159/tailwind-react-native-classnames#137] [Q&A]
I found that I can use tw.style() in the StyleSheet as below, but breakpoints do not seem to work. I have tried a number of configurations to no avail. 
Are the breakpoint conditions available anywhere such as possibly { tw.sm && 'h-12' }?

const styles = StyleSheet.create({
    navContainer: tw.style(
      'flex-row',
      'w-full',
      'bg-slate-700',
      'h-12',
      'sm:h-24',
    ),
})

This did work:
style={ntt()} 
const ntt = () => tw`font-bold text-black text-sm sm:text-3xl`;

**Comment by jaredh159** [comment]
Yeah, I'm pretty sure you won't be able to get it to work.  That's because the `tw` function needs to be connected to a source of runtime device information, by using the `useDeviceContext()` _hook_, as described in the readme.  Because it's a React hook, the call to that needs to be placed _inside a function component_. 

So, what's happening with your code is that the call to `Stylesheet.create()` is getting evaluated before the `tw` function can get connected to the information it needs to evaluate breakpoints.
